### DISCUSSION TITLE
能不能搭建自己的服务器，登录就同步词库

### DISCUSSION THREAD

**Discussion by zhangye151** [rime/home#1476] [Ideas]
能不能搭建自己的服务器，登录就同步词库，宝塔、DOCKER 之类的很方便

**Comment by liuliping0315** [comment]
我之前想尝试onedrive，但是有点儿奇葩。我倒是希望能直接用webdav服务，webdav可以有很多在线供应商，技术水平高的话，自己独立搭建也可以。

**Comment by zjsailor** [comment]
用公共的同步服务器就足够了，没必要自己搭，要自己搭的话，nextcloud, 也是可以，免费的有亿方云，坚果云，百度云，等等。甚至自己用nas，加密同步也行。用久了就发现，词库都不太需要变。我用了很多年了，现在都不用配置了，新电脑一上手，整个目录的配置文件拷过去，重新布署，马上就能用了。

**Reply by Pathsis** [reply]
每次开机自动维护是不是意味着自动同步？

**Reply by zjsailor** [reply]
见rime说明，同步的话，配置文件是单向同步，词库之类的可以双向同步，这是为了防止配置文件出错。大部分情况：在新电脑上安装软件后，将所有文件拷贝过来覆盖用户文件夹即可上手使用，平时的同步，更多是指词库同步。其他配置几乎不用修改。使用的更久之后，会发现，词库都不变了，不需要同步了。只要电脑坏了，安装新设备时，才需要同步一下。此时rime的配置文件更多是备份，而不是实时同步。

**Comment by flakrp** [comment]
我用群晖在drive底下配置好一个用户词库和双拼方案，然后新的系统我就装drive然后指向那个文件夹就可以，只不过mac和linux的话需要做不同的配置，稍微麻烦点，不过也可以实现跨设备共享
